src/jupyter_nbextensions_configurator/__init__.py

```diff
@@ -41,6 +41,12 @@ def get_configurable_nbextensions(
                 'Jupyter Notebook Extension'
         - Main: relative url of the nbextension's main javascript file
     """
+    # deduplicate nbextensions path, see
+    #   github.com/Jupyter-contrib/jupyter_nbextensions_configurator/issues/25
+    seen = set()
+    nbextension_dirs = [
+        d for d in nbextension_dirs if not (d in seen or seen.add(d))]
```

Comment:
yeah, I think you're right, it's a bit opaque, and given the small number of elements in the list, we needn't worry about execution speed anyway. I'll push an amended version now...